community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/OffHeapNumberArray.java

```diff
@@ -52,7 +52,11 @@ public void close()
     {
         if ( !closed )
         {
-            UnsafeUtil.free( address );
+            if ( length > 0 )
```

Comment:
lovely, lets have a test for this case? when we creating OffHeapNumberArray of 0 size and closing it without and we do not fail trying to free 0 address